src/devtools/views/actionList/index.js

```diff
@@ -1,13 +1,33 @@
 import React from 'react'
+import withSend from 'microcosm/addons/with-send'
 import Item from './item'
 import StickyBar from './stickyBar'
 import css from './actionList.css'
 
+const ItemWrapper = withSend(function({ action, isFocused, head, send }) {
+  return (
+    <Item
+      action={action}
+      isFocused={isFocused}
+      head={head}
+      key={action.id}
+      onClick={() => send('detail', isFocused ? head : action.id)}
+    />
+  )
+})
```

Comment:
@nhunzaker what are your thoughts on this? The `Application` presenter is pulling out everything from the model, and then various view components just pass down that data through the ranks. At what point do you start to rethink about the way these components live such that they become their own presenters that pull out the pieces of state only they care about?